test/allowlist_for_publicAPI.json

```diff
@@ -12,7 +12,7 @@
     "torch.nn.intrinsic.quantized.modules": "torch.ao.nn.intrinsic.quantized.modules",
     "torch.nn.intrinsic.quantized.modules.bn_relu": "torch.ao.nn.intrinsic.quantized.modules.bn_relu",
     "torch.nn.intrinsic.quantized.modules.conv_relu": "torch.ao.nn.intrinsic.quantized.modules.conv_relu",
-    "torch.nn.intrinsic.quantized.modules.linear_relu": "torch.ao.nn.intrinsic.quantized.modules.linear_relu",
+    "torch.nn.intrinsic.quantized.modules.linear_activation": "torch.ao.nn.intrinsic.quantized.modules.linear_activation",
```

Comment:
is this change still valid? we don't have `torch.nn.intrinsic.quantized.modules.linear_activation` now

Reply:
It's still needed whereas the file name id changed:
```
"torch.nn.intrinsic.quantized.modules.linear_relu": "torch.ao.nn.intrinsic.quantized.modules.linear_activation",
```

Reply:
I reverted the file name change due to CI checks failure.